packages/cli/src/commands/generate/cell/cell.js

```diff
@@ -66,22 +67,23 @@ export const files = async ({
     (isWordPluralizable(cellName) ? isPlural(cellName) : options.list) ||
     options.list
 
+  // needed for the singular cell GQL query find by id case
+  try {
+    model = await getSchema(pascalcase(singularize(cellName)))
+    idType = getIdType(model)
+    mockIdValue = idType === 'String' ? "'42'" : 42
+  } catch {
+    // eat error so that the destroy cell generator doesn't raise when try to find prisma query engine in test runs
```

Comment:
```suggestion
    // Eat error so that the destroy cell generator doesn't raise an error
    // when trying to find prisma query engine in test runs.

```